local-oldest-requirements.txt

```
@@ -1 +1 @@
-acme[dev]==0.26.0
+acme[dev]==0.29.0
```

Ok FINAL ANSWER really this time: it DOES have to be `-e acme[dev]` and then changed back after the release. I know, I know.
```suggestion
-e acme[dev]
```

---

:)